libmamba/include/mamba/core/fsutil.hpp

```diff
@@ -132,6 +132,21 @@ namespace mamba
         }
 
     }  // namespace path
+
+    namespace mamba_fs
+    {
+        // Like std::rename, but works across file systems
```

Comment:
What does "across file systems" means here?

Reply:
Or did you mean that this is more like a `move`?

Reply:
I think the issue is that sometimes `tmp` is on a different filesystem partition and then the `rename` doesn't work as one needs to copy the file. Maybe there is something builtin to do `rename_or_move`?

Reply:
Other ways would be to get into platform-specific apis, so I think it's fine here. Although indeed the name is not clear to me and some clarification on when/why it should be used in the comments would help.

Reply:
Agreed, will add!